application/Modules/Mouse/Producers/MouseClickEventProducer.cs

```diff
@@ -1,16 +1,85 @@
-﻿using System.ComponentModel.Composition;
+using System.Windows;
+using System.Windows.Input;
 using MORR.Modules.Mouse.Events;
 using MORR.Shared.Events.Queue;
+using MORR.Shared.Utility;
 
 namespace MORR.Modules.Mouse.Producers
 {
     /// <summary>
     ///     Provides a single-writer-multiple-reader queue for MouseClickEvent
     /// </summary>
-    [Export(typeof(MouseClickEventProducer))]
-    [Export(typeof(IReadOnlyEventQueue<MouseClickEvent>))]
     public class MouseClickEventProducer : DefaultEventQueue<MouseClickEvent>
```

Comment:
This does not appear to generate click events in the non-client area of windows (e.g. the title bar). Could you check if this is also a problem when using the low-level hook?

Reply:
Karl is on it, waiting on his further information